lang/c/src/schema_equal.c

```diff
@@ -50,10 +50,18 @@ schema_record_equal(struct avro_record_schema_t *a,
 		}
 		if (!avro_schema_equal(fa.f->type, fb.f->type)) {
 			/*
-			 * They have fields with different schemas 
+			 * They have fields with different schemas
 			 */
 			return 0;
 		}
+                /*
+                 * TODO: Do we need to check that the default values
+                 * are equal?  If so, then
```

Comment:
Per [the spec](http://avro.apache.org/docs/current/spec.html#Parsing+Canonical+Form+for+Schemas), no.  Default values are not included when canonicalizing a schema, so they can't be included when checking for equality, either.